code/game/objects/items/dualsaber.dm

```diff
@@ -29,6 +29,7 @@
 	resistance_flags = FIRE_PROOF
 	wound_bonus = -10
 	bare_wound_bonus = 20
+	demolition_mod = 1.5 //1.5x damage to objects, robots, etc.
```

Comment:
Shouldn't the base variable declaration just describe what the demolition mod does itself

Reply:
probably, but the redundancy can't hurt, right?